addons/sale_project/models/sale_order_line.py

```diff
@@ -271,8 +271,8 @@ def _timesheet_create_task_prepare_values(self, project):
             title = self.product_id.name
             description = '<br/>'.join(sale_line_name_parts)
         else:
-            title = sale_line_name_parts[0]
-            description = '<br/>'.join(sale_line_name_parts[1:])
+            title = " - ".join(sale_line_name_parts[:2])
```

Comment:
We expect to have just the first line of description when there is a description set otherwise we display the product name. 🙂 

Can you do the change accordingly? 

Thanks in advance! 🙂